x/bank/module.go

```diff
@@ -248,12 +250,16 @@ func ProvideModule(in ModuleInputs) ModuleOutputs {
 		authority = authtypes.NewModuleAddressOrBech32Address(in.Config.Authority)
 	}
 
+	// add the module name to the logger
+	logger := in.Logger.With("module", "x/"+types.ModuleName)
```

Comment:
For making legacy app.go still nice to use, I think this should be done in the keeper.
Otherwise, it'll all nice in app_v2, but in app.go you need for each keeper to precise `logger.With`

Reply:
ah, you are right. There was also a Logger() method in the keeper but I don't think we need that